code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -269,7 +272,7 @@ Difficulty: Medium
 		sleep(0.5)
 	
 	// Ash drake flies onto its target and rains fire down upon them
-	var/descentTime = 7;
+	var/descentTime = 10;
```

Comment:
this probably wasn't a problem given that it was at 0.5 before

Reply:
This is increasing the time it takes for the dragon to hit the ground once he flies above you, what are you talking about?